faker/providers/address/th_TH/__init__.py

```diff
@@ -20,13 +20,52 @@ class Provider(AddressProvider):
         ("{{street_address}} {{tambon}} จ.{{province}} {{postcode}}", 15),
         ("{{street_address}} อ.{{amphoe}} จ.{{province}} {{postcode}}", 15),
         ("{{street_address}} ต.{{tambon}} จ.{{province}} {{postcode}}", 15),
+        ("{{street_address}} อำเภอ{{amphoe}} จังหวัด{{province}} {{postcode}}", 15),
+        ("{{street_address}} ตำบล{{tambon}} อำเภอ{{amphoe}} จังหวัด{{province}} {{postcode}}", 10),
         ("{{street_address}} {{province}} {{postcode}}", 15),
         ("{{street_address}} ต.{{tambon}} อ.{{amphoe}} {{province}}", 15),
         ("{{street_address}} ต.{{tambon}} อ.{{amphoe}} จ.{{province}}", 15),
+        ("{{street_address}} ตำบล{{tambon}} จังหวัด{{province}} {{postcode}}", 10),
         ("{{building_number}} ต.{{tambon}} อ.{{amphoe}} {{province}} {{postcode}}", 10),
         ("{{building_number}} หมู่บ้าน{{first_name}} {{amphoe}} {{province}} {{postcode}}", 10),
     ))
 
+    city_formats = ("{{city_name}}",)
+    cities = (
```

Comment:
can you add the source of this list as a comment above this line?

Reply:
Thank you. Added.